packages/react-components/src/index.ts

```diff
@@ -1,6 +1,7 @@
 import './index.scss';
 
 export { DesignTokens } from './themes/designTokens';
+export { SpacingTokens } from './spacing/spacingTokens';
```

Comment:
Enumerations (even realized as consts) should have a singular form.
```suggestion
export { SpacingToken } from './foundation/spacing-tokens';
```